### PR TITLE
Refactor website

### PR DESCRIPTION
**Pull request by husky-parul**
- improve navigation
- suggest recommended installation strategies
- add search

### CODE REVIEW

File: docs/installation/local-cluster.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    containerPath: /proc-host
+  - hostPath: /usr/src
+    containerPath: /usr/src
+```
+
+We can then spin up a cluster with:
+
+```sh
+export $CLUSTER_NAME="my-cluster"  # we can use the --name flag to override the name in our config
+kind create cluster --name=$CLUSTER_NAME --config=./local-cluster-config.yaml
```
Comment (husky-parul) [inline docs/installation/local-cluster.md:30]: Is this doc updated? I don't think we do local cluster set up using yaml anymore.
@SamYuan1990

File: docs/installation/strategy.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Installation Strategies
+
+While you are free to explore any deployments but the recommended stratgies are :
+
+| *OCP 4.13*      | *Microshift*     | *RHEL*  |  *ROSA* | *Kind* |
+| ------------- | -------------  | ----- | ----- | ----|
+| kepler-operator | Manifests  | RPM | Manifests | Helm Charts, Manifests, kepler-operator
```
Comment (husky-parul) [inline docs/installation/strategy.md:7]: @SamYuan1990 can you verify this table? Is it possible to use helm charts with microshift?

File: docs/installation/local-cluster.md (hunk 1 above)
Comment (husky-parul) [reply docs/installation/local-cluster.md:30]: I use `make cluster-up CLUSTER_PROVIDER='kind' CI_DEPLOY=true GRAFANA_ENABLE=true` to start the local cluster. What is the recommended option for kind?

Comment (SamYuan1990) [reply docs/installation/local-cluster.md:30]: I will check and fix in another PR if needed.

File: docs/installation/strategy.md (hunk 2 above)
Comment (SamYuan1990) [reply docs/installation/strategy.md:7]: I suppose as we have microshift and kind  in our CI, we can ensure those works. But I have no idea with OCP, RHEL, ROSA...
I suppose helm works with microshift
https://github.com/sustainable-computing-io/kepler-helm-chart/actions/runs/5487590207/jobs/9999243718#step:4:567
https://github.com/sustainable-computing-io/kepler-helm-chart/actions/runs/5487590207/jobs/9999243718#step:8:7

@husky-parul , @rootfs please let me know if anything further I can help with.

**Review by rootfs — APPROVED** [review]

**Pull request merged**